src/main/frontend/common.css

```diff
@@ -697,20 +697,6 @@ svg.tip {
   color: var(--ls-active-primary-color);
 }
 
-/* endregion */
-
-/* region FIXME: CodeMirror override (?) */
-.CodeMirror pre.CodeMirror-line,
-.CodeMirror-scroll,
-.CodeMirror-sizer,
-.CodeMirror-gutter,
-.CodeMirror-gutters,
-.CodeMirror-linenumber {
-  font-size: 14px;
-}
-
-/* endregion */
```

Comment:
At least 1 `endregion` should be kept.

Reply:
@andelf, I've re-added it. It makes sense to have both the opening and closing `endregion` comments for better code organization and clarity. Thank you for the feedback!